ui/src/styles.css

```diff
@@ -471,12 +471,13 @@ body .p-menu .p-menuitem-link {
   align-items: center;
 }
 
-/* React Select styling to increase size of underlying input tag */
-.css-b8ldur-Input {
-  width: 100%;
-  z-index: 1;
-}
-
-.css-b8ldur-Input :is(div, input) {
-  width: 100%;
+/* React Select styling to increase size of underlying input tag
```

Comment:
ReactSelect needs to be updated in three places in order to get the input to size how we would like. This updates the two inner items. These are labelled "B" and "C" in this diagram.